canonical/lxd-ui
# feat: use TablePagination component from react-components for paginating tables [WD-8209]

## Done

- removed `Pagination.tsx`, `pagination.tsx` and `_pagination.scss`.
- The above component and styles have been upstreamed to react-components as `TablePagination`.
- Applied `TablePagination` component across the app where previously the `Pagination` component was used together with the `usePagination` hook.
- updated `ScrollableTable` to use the id of a child table element for updating its height in the DOM.

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @lorumic or @edlerd for access.
    - With a local copy of this branch, run as described [here](../HACKING.md#setting-up-for-development).
2. Perform the following QA steps:
    - check if pagination works as expected across the app. It's the same component, so checking in one place is sufficient. Suggest checking the instance list as it has the most complex pagination use case.
